docs/excel/excel-add-ins-error-handling.md

```diff
@@ -47,12 +47,15 @@ When an Excel JavaScript API request fails to run successfully, the API returns
 
 The following table is a list of errors that the API may return.
 
-|error.code | error.message |
+|error.code | Error Message |
 |:----------|:--------------|
 |`AccessDenied` |You cannot perform the requested operation.|
 |`ActivityLimitReached`|Activity limit has been reached.|
 |`ApiNotAvailable`|The requested API is not available.|
+|`ApiNotFound`|The API you are trying to use could not be found. It may be available in a newer version of Excel.|
+|`BadPassword`|The password you supplied is not correct.|
 |`Conflict`|Request could not be processed because of a conflict.|
+|`ContentLengthRequired`|A Content-Length header is missing.|
```

Comment:
Should "Content-Length" be capitalized like that? I'm not familiar with that term.

Reply:
I think it refers to this: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Length. In our docs, the entity is referred to as a `Content-length` HTTP header (https://docs.microsoft.com/en-us/dotnet/api/system.net.httpwebrequest.contentlength?view=netcore-3.1). What do you think of this as the updated error message: 

```suggestion
|`ContentLengthRequired`|A `Content-length` HTTP header is missing.|
```